### ISSUE TITLE
[FORMATTING] Error: Cannot find module 'src/languages/bigquery.formatter'

### ISSUE THREAD

**Issue by JavaHello** [sql-formatter-org/sql-formatter#234]
**Describe the bug**
```txt
Error: Cannot find module 'src/languages/bigquery.formatter'
```

**Actual Output**

```txt
node:internal/modules/cjs/loader:942
  throw err;
  ^

Error: Cannot find module 'src/languages/bigquery.formatter'
Require stack:
- /opt/homebrew/lib/node_modules/sql-formatter/lib/sqlFormatter.js
- /opt/homebrew/lib/node_modules/sql-formatter/lib/index.js
- /opt/homebrew/lib/node_modules/sql-formatter/bin/sql-formatter-cli.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:939:15)
    at Module._load (node:internal/modules/cjs/loader:780:27)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/opt/homebrew/lib/node_modules/sql-formatter/lib/sqlFormatter.js:8:40)
    at Module._compile (node:internal/modules/cjs/loader:1105:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1159:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Module._load (node:internal/modules/cjs/loader:827:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/opt/homebrew/lib/node_modules/sql-formatter/lib/sqlFormatter.js',
    '/opt/homebrew/lib/node_modules/sql-formatter/lib/index.js',
    '/opt/homebrew/lib/node_modules/sql-formatter/bin/sql-formatter-cli.js'
  ]
}

Node.js v18.3.0

```

**Usage**

1. `npm --location=global install sql-formatter`
1. `sql-formatter -h`

**Comment by nene** [comment]
Damn... looks like the switch to using absolute paths `src/languages/bigquery.formatter` messed up the library.

Thinking how to fix it...

**Comment by rogerfar** [comment]
Getting same error, but for more formatters:
```
ERROR in ../../node_modules/sql-formatter/lib/sqlFormatter.js 8:39-82
Module not found: Error: Can't resolve 'src/languages/bigquery.formatter' in 'client\node_modules\sql-formatter\lib'

ERROR in ../../node_modules/sql-formatter/lib/sqlFormatter.js 10:33-71
Module not found: Error: Can't resolve 'src/languages/db2.formatter' in 'client\node_modules\sql-formatter\lib'

ERROR in ../../node_modules/sql-formatter/lib/sqlFormatter.js 12:35-74
Module not found: Error: Can't resolve 'src/languages/hive.formatter' in 'client\node_modules\sql-formatter\lib'

ERROR in ../../node_modules/sql-formatter/lib/sqlFormatter.js 14:38-80
Module not found: Error: Can't resolve 'src/languages/mariadb.formatter' in 'client\node_modules\sql-formatter\lib'

ERROR in ../../node_modules/sql-formatter/lib/sqlFormatter.js 16:36-76
Module not found: Error: Can't resolve 'src/languages/mysql.formatter' in 'client\node_modules\sql-formatter\lib'

ERROR in ../../node_modules/sql-formatter/lib/sqlFormatter.js 18:35-74
Module not found: Error: Can't resolve 'src/languages/n1ql.formatter' in 'client\node_modules\sql-formatter\lib'

ERROR in ../../node_modules/sql-formatter/lib/sqlFormatter.js 20:36-76
Module not found: Error: Can't resolve 'src/languages/plsql.formatter' in 'client\node_modules\sql-formatter\lib'

ERROR in ../../node_modules/sql-formatter/lib/sqlFormatter.js 22:41-86
Module not found: Error: Can't resolve 'src/languages/postgresql.formatter' in 'client\node_modules\sql-formatter\lib'

ERROR in ../../node_modules/sql-formatter/lib/sqlFormatter.js 24:39-82
Module not found: Error: Can't resolve 'src/languages/redshift.formatter' in 'client\node_modules\sql-formatter\lib'

ERROR in ../../node_modules/sql-formatter/lib/sqlFormatter.js 26:36-76
Module not found: Error: Can't resolve 'src/languages/spark.formatter' in 'client\node_modules\sql-formatter\lib'

ERROR in ../../node_modules/sql-formatter/lib/sqlFormatter.js 28:37-78
Module not found: Error: Can't resolve 'src/languages/sqlite.formatter' in 'client\node_modules\sql-formatter\lib'

ERROR in ../../node_modules/sql-formatter/lib/sqlFormatter.js 30:34-72
Module not found: Error: Can't resolve 'src/languages/sql.formatter' in 'client\node_modules\sql-formatter\lib'

ERROR in ../../node_modules/sql-formatter/lib/sqlFormatter.js 32:35-74
Module not found: Error: Can't resolve 'src/languages/tsql.formatter' in 'client\node_modules\sql-formatter\lib'
```

**Comment by nene** [comment]
Should be fixed now in 6.1.5.

Added two plugins:

- `module-resolver` to fix imports in .js files produced by Babel compilation
- `@zerollup/ts-transform-paths` to fix imports in .d.ts files produced by TypeScript compilation
